recipes/dismod-at/recipe.yaml

```diff
@@ -0,0 +1,72 @@
+# conda-verify warnings:
+#
+schema_version: 1
+
+context:
+  name: dismod-at
+  version: 2025.0.2
+
+package:
+  name: ${{ name }}
+  version: ${{ version }}
+
+source:
+  #
+  url: https://github.com/bradbell/dismod_at/archive/${{ version }}.tar.gz
+  sha256: c511a4d59114960940f7d5f83c3bf84268cf3b76f123557ba21514e4e376962d
+  #
+  patches:
+    # dismodat.py: set python executable so works with conda build environment.
+    - ${{ version }}.patch
+
+build:
+  # first build on this version
+  number: 0
+
+requirements:
+  build:
+    - ${{ compiler('c') }}
+    - ${{ stdlib('c') }}
+    - ${{ compiler('cxx') }}
+    - cmake
+    - ninja
+  host:
+    - python
+    - pkg-config
+    - cppad-mixed
+    - sqlite
+    - pip
+    - eigen
+    - numpy
+    - matplotlib
+    - scipy
+    - setuptools
+  run_exports:
+    - ${{ pin_subpackage(name, upper_bound='x.x') }}
+
+tests:
+  - script:
+      interpreter: python
+      file: run_test.py
+    requirements:
+      run:
+        - python
+        - numpy
+        - matplotlib
+        - scipy
+        - dismod-at
```

Comment:
```suggestion
```

The installed package is already in the environment. Asking for it as a run requirement will try to install it.